### PR TITLE
Adding Multi channel modems

### PR DESCRIPTION
**Pull request by AnneThKu**
Below changes are done:
- Added `Acoustic Data Acquisition system` in `Products` page.
- Updated `Water Quality Monitoring` to `Subnero Water Assessment Network`.
- Added `aRecorder` and `multi-channel modem` to DAQ category.
- Added `multi-channel modem` to Silver edition section as well.
- Added a line for availability of customised products in the modems page.

### CODE REVIEW

**Comment by AnneThKu** [comment]
Noting down the broken links.
- 2018-05-27-UnetStack-Newsletter.md
  - http://www.oceans18mtsieeekobe.org/social-events/
  - http://www.oceans18mtsieeekobe.org/tutorial-information/
- 2019-06-11-UnetStack-Newsletter.md
  - http://www.oceans19mtsieeemarseille.org/ 
- 2019-10-15-Subnero-expands-in-china.md
  - http://www.smartseatech.com/en/index.aspx 
- 2019-10-29-Subnero-at-iOcean-Sanya.md
  - http://www.cso.org.cn/ggxx/2019/0926/2816.html 
  - https://singapore20.oceansconference.org/

**Review by notthetup — APPROVED** [review]

**Comment by notthetup** [comment]
@AnneThKu I've raised an issue for broken links https://github.com/subnero1/subnero1.github.io/issues/119 

Maybe you can add the ones you found to the same one.

**Review by notthetup — APPROVED** [review]

File: products/accessories.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  <div class='four spacing'></div>
+</div>
+
+<!--<div class='full' style='background:#F5F5F5'>-->
+<div class='cGBxoB'>
+<section class='bsPRnx'>
+	<h1 class='thin' style='text-align: center'>Optional Accessories and Upgrades</h1>
+	<div class='eyXpDN'>
+		<div class='cmXrEt'>
+			<!-- <h1 class='thin' style='text-align: center'>Platinum Edition</h1> -->
```
Comment (notthetup) [inline products/accessories.md]:
```suggestion
```

File: products/accessories.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  <div class='row'>
+    <div class='large-12 columns'>
+      <!-- {% include section-header.html title=page.title tagline=page.tagline color=page.title_color class="big" %} -->
+    </div>
+  </div>
+  <div class='four spacing'></div>
+  <div class='four spacing'></div>
+</div>
+
+<!--<div class='full' style='background:#F5F5F5'>-->
```
Comment (notthetup) [inline products/accessories.md]:
```suggestion
```

**Comment by manuignatius** [comment]
This branch is still WIP, the PR was raised a bit too early. Will re-request for a review once the updates are done.

**Comment by notthetup** [comment]
Converted to draft. Convert to "ready" when you're ready for review @manuignatius

**Review by notthetup — APPROVED** [review]

**Comment by manuignatius** [comment]
Review required
- All pages under "Wireless Networked Communications" page.
- Changes
   - Updated platinum edition and silver edition modems pages
   - Updated SC from surface to standalone
   - Removed node configuration
   - Added multi-channel modems
   - Added optional upgrades and accessories section
   - Updated modem brochure and technical specs

**Comment by AnneThKu** [comment]
Travis failing as the below mentioned links are broken:
Node Modem taking the plunge (30-Mar-2020) - https://sites.google.com/edu.haifa.ac.il/anl/staff
IoT Weekly Podcast (19-Jan-2018) - https://gearforgesoftware.com/iot-weekly-podcast/iot-weekly-episode-8-podcast/
Robot SWANs to monitor water quality in Singapore (15-Jan-2018) - https://www.channelnewsasia.com/404

**Comment by notthetup** [comment]
@AnneThKu Can we link to the rendered PDFs/PNGs of the Internet archive versions for those links (if available)?

**Comment by notthetup** [comment]
Here is one.. web archive org_web_20201220173152_https___www channelnewsasia com_news_singapore_robot-swans-reservoirs-monitor-water-quality-pub-9861686_

**Comment by notthetup** [comment]
1. This banner is very strange. The white space in the middle with the modems all the way to the edge looks like something is broken..

Screen Shot 2021-03-16 at 21 14 03

 2. Instead of just saying "D", we can say `D (IEC R20)` which is the official standard name for D-cells
Screen Shot 2021-03-16 at 21 14 33

3. These random stock photos of storage mediums look ugly. If the photos of SDCards are repetitive, we can just put one photo on the page.
Screen Shot 2021-03-16 at 21 15 33

4. Also the mouse over blurring+zooming effect on all the photos in that array makes them look worse.

5. Strange grey line below the photo of the platinum edition modem. Also, why is it horizontal?
Screen Shot 2021-03-16 at 21 16 13

6. The ziplock with handwritten stuff looks very unprofessional. We should print a label or not have any label.
Screen Shot 2021-03-16 at 21 16 42

7. What is this photo supposed to be?
Screen Shot 2021-03-16 at 21 25 55

8. This array has repeated photos..
Screen Shot 2021-03-16 at 21 26 01

9. Some sections have +/- signs on the left and some on the right. Some can be folded, some can't. We should be consistent.
Screen Shot 2021-03-16 at 21 30 09 
Screen Shot 2021-03-16 at 21 14 33

10. Some pages have a pink? background, others are white. We should be consistent.

File: _includes/collapse.html [hunk 3]
```diff
@@ -0,0 +1,20 @@
+<!-- Start Collapse.html -->
```
Comment (notthetup) [inline _includes/collapse.html]: Can we use a pure-css based collapsible instead?

https://www.digitalocean.com/community/tutorials/css-collapsible

File: _posts/2020-10-12-Global-Oceans-2020.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
+---
```
Comment (notthetup) [inline _posts/2020-10-12-Global-Oceans-2020.md:1]: Why are all the pulse posts a part of this pull request?

File: products/accessories.md [hunk 5]
```diff
@@ -0,0 +1,177 @@
...
+  <div class='four spacing'></div>
+</div>
+
+<div class='cGBxoB'>
+<section class='bsPRnx'>
+	<h1 class='thin' style='text-align: center'>Optional Accessories and Upgrades</h1>
+	<div class='eyXpDN'>
+		<div class='cmXrEt'>
+			<h1 class='thin' style='text-align: center'>Battery Packs</h1>
+			<ul class="gfXsQG">
```
Comment (notthetup) [inline products/accessories.md:27]: What does this class `gfXsQG` do?

**Comment by manuignatius** [comment]
1. Will update the banner to reduce the white space
2. Updated D to D (IEC R20)
3. Will need @AnneThKu 's help with this. As of now, there is a limitation of having 4 images in all pages. Hence the random stock images. Would be great if we do not have such a limitation, not sure how hard is it in html/css/javascript. Seems pretty hard.
4. Needs @AnneThKu  to fix this
5. The grey line is the line that comes during mouse over? Image can be updated to vertical, but would like to stick with 3 images if we can have different number of images in a page
6. Will remove the zip lock from the image with only clamps in there
7. Right angled bulk head connector (similar image in the original subconn product page). We can consider changing it to a right angled connector with cable connected. Again, would be great if we can just have 2 images in this page.
8. In general, since there is a constraint of exact 4 images, some of the images are repeated. If we can fix to any number of images between 1 - 4, it will be easier
9. The + sign is for @AnneThKu  to add folding to the page. I think these are pending. If we are not able to do it (seems it is difficult), I will remove those '+' signs and keep all the text visible on page load (which ideally I would like to avoid)
10. The modem product pages are all "pink/grey" bg but the "accessories" pages are all white based on the template. @AnneThKu can you help update all accessories page to "pink/grey"?

File: products/accessories.md [hunk 6]
```diff
@@ -0,0 +1,177 @@
...
+				<li class="fuqHMA">
```
Comment (notthetup) [inline products/accessories.md:28]: Same with class `fuqHMA`?

File: products/batterypack2.md [hunk 7]
```diff
@@ -0,0 +1,113 @@
...
+banner : images/banner-battery.jpg
+thumbnail: images/accessories-batterypack02.jpg
+categories: accessories
+excerpt: Underwater battery packs for deployment of Subnero underwater modems.
+---
+
+<div class='full tall' style='background-image: url({{site.baseurl}}/{{page.banner}});'>
+  <div class='row'>
+    <div class='large-12 columns'>
+      <!-- {% include section-header.html title=page.title tagline=page.tagline color=page.title_color class="big" %} -->
```
Comment (notthetup) [inline products/batterypack2.md]:
```suggestion
```

**Comment by notthetup** [comment]
> The grey line is the line that comes during mouseover? 

I see the grey line under some images all the time. Something about screen size?

**Comment by notthetup** [comment]
I tried with 3 images, and I get this. Seems OK.
 
Screen Shot 2021-03-16 at 21 52 04

**Comment by manuignatius** [comment]
> I tried with 3 images, and I get this. Seems OK.
> 
> Screen Shot 2021-03-16 at 21 52 04

The size of the images changes if the number of images are less. Try resizing the page or a different screen size, you will be able to see it. Basically, the thumbnails starts to become bigger and bigger.

File: products/accessories.md (hunk 6 above)
Comment (manuignatius) [reply products/accessories.md:28]: Not sure I understand :(

**Comment by manuignatius** [comment]
1. Updated battery image

Will need @AnneThKu help with
- Fix collapsing in the modem pages
- Fixing bg color in accessories page to match with modem page
- Having random number of images in accessories page without changing size

Once these are done, I can fix the rest of the comments.

**Review by mchitre — APPROVED** [review]
@manuignatius
Overall much better than what we have, so I recommend going ahead with this.

A critique, however, is that the content doesn't seem to flow smoothly, in general. While nothing incorrect with the text, the overall pages feel like disconnected text blurbs. At some point, we might need a more holistic look at the overall flow of the site.

For example, if you look at the multichannel modem page, it starts with "Subnero underwater modems provide options for customization and extension at many levels." But this is a standard configuration, so there's no customization involved! Then after key features, you have these 3 inset boxes which feel like they're meant to clickable to get more info, but they're not. And they just repeat what the last 3 bullets of features say.

File: _includes/collapse.html (hunk 3 above)
Comment (AnneThKu) [reply _includes/collapse.html]: Changed to css-collapsible in https://github.com/subnero1/subnero1.github.io/pull/120/commits/aba146c0e2624dd6af0b635d60c3f3681ffe30bd and  https://github.com/subnero1/subnero1.github.io/pull/120/commits/bab7b4bc084ab549d7cccbfe82cb8f0c43f73534.

File: products/accessories.md (hunk 6 above)
Comment (notthetup) [reply products/accessories.md:28]: It's best practice to give CSS classes names that make sense. For example `class="col-xs-6` which implies this is a `column` of `xs` size and 6 points in width. But `fuqHMA` doesn't mean anything.

Comment (AnneThKu) [reply products/accessories.md:28]: These classes were taken from the reference provided during the revamping of the product listing page and were used as it is. I have raised an issue to rename all the classes (https://github.com/subnero1/subnero1.github.io/issues/148).

**Comment by AnneThKu** [comment]
> @AnneThKu Can we link to the rendered PDFs/PNGs of the Internet archive versions for those links (if available)?

When the archived version is added as link, the image opens as zoomed-in, and once the user clicks, it zooms out to a size that we can read. So, added as image in https://github.com/subnero1/subnero1.github.io/pull/120/commits/72d77b998fcbfe64b60297128564478ecd19e824. Please check and let me know if this looks ok.

What can be done for the remaining links?
- Can we use this link https://sites.google.com/edu.haifa.ac.il/anl/personnel for `Node Modem taking the plunge (30-Mar-2020) - https://sites.google.com/edu.haifa.ac.il/anl/staff`?

>Will need @AnneThKu help with
Fix collapsing in the modem pages
Fixing bg color in accessories page to match with modem page
Having random number of images in accessories page without changing size

All the above mentioned issues are fixed. Please help to check.

**Comment by manuignatius** [comment]
> @manuignatius
> Overall much better than what we have, so I recommend going ahead with this.
> 
> A critique, however, is that the content doesn't seem to flow smoothly, in general. While nothing incorrect with the text, the overall pages feel like disconnected text blurbs. At some point, we might need a more holistic look at the overall flow of the site.
> 
> For example, if you look at the multichannel modem page, it starts with "Subnero underwater modems provide options for customization and extension at many levels." But this is a standard configuration, so there's no customization involved! Then after key features, you have these 3 inset boxes which feel like they're meant to clickable to get more info, but they're not. And they just repeat what the last 3 bullets of features say.

Updated the multi-channel modem page to address your comments.

Yes, I agree, once we publish these (and may be after the next couple of minor updates), we need a holistic review of our entire website.

**Comment by manuignatius** [comment]
> > @AnneThKu Can we link to the rendered PDFs/PNGs of the Internet archive versions for those links (if available)?
> 
> When the archived version is added as link, the image opens as zoomed-in, and once the user clicks, it zooms out to a size that we can read. So, added as image in [72d77b9](https://github.com/subnero1/subnero1.github.io/commit/72d77b998fcbfe64b60297128564478ecd19e824). Please check and let me know if this looks ok.
> 
> What can be done for the remaining links?
> 
> * Can we use this link https://sites.google.com/edu.haifa.ac.il/anl/personnel for `Node Modem taking the plunge (30-Mar-2020) - https://sites.google.com/edu.haifa.ac.il/anl/staff`?
> 
> > Will need @AnneThKu help with
> > Fix collapsing in the modem pages
> > Fixing bg color in accessories page to match with modem page
> > Having random number of images in accessories page without changing size
> 
> All the above mentioned issues are fixed. Please help to check.

@AnneThKu I tested all the image related comments from @notthetup and all seems to work in my setup. There is only 1 comment pending that I need your help with.

There are some `+` signs in the various modem pages (only in modem pages) which are meant to be made to collapsible. Can you apply the collapse feature to one of the pages so that I can adapt it to the rest of the pages? Pages are:
- wnc-m25mpe3.md
- wnc-m25mps3.md
- wnc-m25mrs3.md
- wnc-m25mse3.md
- wnc-m25mss3+xch.md
- wnc-m25mss3.md

**Comment by Florie25** [comment]
@manuignatius , here are my comments:
**Landing page:**
- pictures of Platinum edition modems should be centred.
- could we replaced the picture of the multichannel modem with a picture of a full modem. The current picture makes the cut really stand out.
- Can we change the banner to get rid of the node configuration?

**Primary Battery pack page:**
- Consider "on-site' instead of "in-field" replaceable
- typo under product information "rechargeable"

**Comment by manuignatius** [comment]
@Florie25 Addressed all your comments
- Centred the platinum modems (good catch)
- New image for multi-channel modem
- New banner image for modem page - removed node configuration
- Updated to "on-site"
- Fixed typo

**Review by manuignatius — APPROVED** [review]

**Review by notthetup — APPROVED** [review]
Looks good. Only one comment, the

> CONTACT US FOR A QUOTE

on each of the product pages seems very awkwardly placed between the product description and specs. Maybe another way to position that would be nice?

**Comment by manuignatius** [comment]
Good catch @notthetup , that was supposed to be a button, the text was only a placeholder. Let me see how I can update it.

**Pull request merged**